### ISSUE TITLE
Add a custom level folder

### ISSUE THREAD

**Issue by Vadru93** [Vadru93/LevelMod#25]
I was thinking we could add a custom level folder.
This folder could auto generate a change level array and we could add Custom Level to change level menu.
This could be used when we are testing maps, or if people wanna add their own custom maps etc.

**Comment by DCxDemo** [comment]
Sure, but it might be tricky.

Can store custom levels in a separate array probably and populate the custom levels menu using data from that array.
Then custom level should provide definition struct to be called by load level func. What should it be? a master qb for all files? a separate qb for each level? a text file? json/xml/ini?

Should also think about corner cases, what if people don't have custom level?
Should test checksums even if levels match. maybe just a warning so you know it's a different version of the level.

**Comment by Vadru93** [comment]
Since we can now send data between server and client we could probably have a check if level don’t exist, download it from host and level checksum should never mismatch because the auto update should always make sure you download the latest version of a level.

**Comment by DCxDemo** [comment]
yeah but it's about custom levels, not the ones included in the mod.
